modules/ROOT/attachments/admin-api.yaml

```diff
@@ -5232,6 +5488,11 @@ tags:
 
       * Start and start broker maintenance
       * Decommission and recommision brokers
+
+  - name: Migrations
```

Comment:
Added the new endpoints under a new "Migrations" section. Does this make sense or should I create something like a "Topics" section or add the endpoints to an existing section instead?